lib/active_model/serializer.rb

```diff
@@ -4,15 +4,15 @@
 require 'active_model/serializer/error_serializer'
```

Comment:
Yes, sure. The name change was only defined as a note in the RFC, should I update the RFC too?